deployments/stacks/dpe-k8s/main.tf

```diff
@@ -56,3 +56,37 @@ module "synapse_dataset_to_crossiant_metadata" {
   public_access = true
   enable_cors = true
 }
+
+module "synapse-webhook-api-gateway" {
+  source = "../../../modules/aws-api-gateway"
+  environment = var.cluster_name == "dpe-k8s-sandbox" ? "sandbox" : "dev"
+  name = "synapse-webhook-api-gateway"
+}
+
+module "synapse-sqs-create-queue" {
+  source = "../../../modules/aws-sqs"
+  environment = var.cluster_name == "dpe-k8s-sandbox" ? "sandbox" : "dev"
+  name = "synapse-sqs-create-queue"
+  aws_account_id = var.aws_account_id
+  aws_region = var.region
+  cluster_oidc_provider_arn = module.sage-aws-eks.cluster_oidc_provider_arn
+  
+  # API Gateway integration
+  enable_api_gateway_integration = true
+  api_gateway_id = module.synapse-webhook-api-gateway.api_id
+  route_path = "/create"
+}
+
+module "synapse-sqs-delete-queue" {
+  source = "../../../modules/aws-sqs"
+  environment = var.cluster_name == "dpe-k8s-sandbox" ? "sandbox" : "dev"
+  name = "synapse-sqs-delete-queue"
+  aws_account_id = var.aws_account_id
+  aws_region = var.region
+  cluster_oidc_provider_arn = module.sage-aws-eks.cluster_oidc_provider_arn
+  
+  # API Gateway integration
+  enable_api_gateway_integration = true
+  api_gateway_id = module.synapse-webhook-api-gateway.api_id
+  route_path = "/delete"
+}

```

Comment:
We don't have a current business need for these SQS queues so I will be deleting these module invocations before merging, but wanted to leave this in as an example to show how I deployed the gateway and queues in the sandbox for testing

Reply:
Once we do the writeup, we can send this out to scidata and ask for some use cases.  Does the queue only work with one project?

Reply:
Yes, each Webhook can only subscribe to one Synapse Entity. You can point multiple Webhooks at a single SQS queue though so you could monitor multiple entities with one queue. The only limit there is 25 Webhooks per user.